build_tools/cmake/build_all.sh

```diff
@@ -23,6 +23,13 @@ IREE_ENABLE_ASSERTIONS="${IREE_ENABLE_ASSERTIONS:-ON}"
 # needed, but some of the deps are too large to enable by default for all
 # developers.
 IREE_TARGET_BACKEND_WEBGPU_SPIRV="${IREE_TARGET_BACKEND_WEBGPU_SPIRV:-ON}"
+# Enable CUDA and HIP/ROCM compiler and runtime by default if not on Darwin.
+# As with WebGPU the dependencies get fetched.
+platform_supported="$(uname | awk '{print ($1 == "Darwin") ? "OFF" : "ON"}')"
```

Comment:
nit: the variable name `platform_supported` doesn't specify _what_ the platform supports. Maybe rename to `is_apple_platform` or something

Reply:
Rename to OFF_IF_DARWIN